gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1033,3 +1030,99 @@ message Shape {
   // The following extension IDs are reserved for private use by any organization.
   extensions 9000 to 9999;
 }
+
+// Describes a stop which is served by trips. All fields are as described in the GTFS-Static specification.
+message Stop {
+  enum WheelchairBoarding {
+    UNKNOWN = 0;
+    AVAILABLE = 1;
+    NOT_AVAILABLE = 2;
+   }
+
+  required string stop_id = 1;
+  optional TranslatedString stop_code = 2;
+  optional TranslatedString stop_name = 3;
+  optional TranslatedString tts_stop_name = 4;
+  optional TranslatedString stop_desc = 5;
+  optional float stop_lat = 6;
+  optional float stop_lon = 7;
```

Comment:
```suggestion
  required TranslatedString stop_name = 3;
  optional TranslatedString tts_stop_name = 4;
  optional TranslatedString stop_desc = 5;
  required float stop_lat = 6;
  required float stop_lon = 7;
```
suggestion: since https://github.com/google/transit/pull/403/files#diff-abc780f2e1f0fd946abf169b0bb5d5ed06c33c695a028c7119043c0584eb7759R56 indicates this is a `location_type` 0 stop, it must have `stop_name`, `stop_lat`, and `stop_lon`.

Reply:
That would be a bad idea because required is forever (see [proto docs](https://protobuf.dev/programming-guides/proto2/#specifying-rules)). If we eventually want to allow more `location_type`, we can't make any of those fields optional.

Reply:
However I see the reference.md doesn't reflect that they should be Required, I'll update this.